.travis.yml

```diff
@@ -27,28 +19,28 @@ install:
  - sudo apt-get install -y xvfb herbstluftwm
 
  # Qt
- - python scripts/install-qt.py
+ - $PYTHON scripts/install-qt.py
 
- # PyTest
- - pip install -U pytest
+ # Pip (3.4 does not have it by default)
+ - wget https://bootstrap.pypa.io/get-pip.py
+ - sudo $PYTHON get-pip.py
 
- # Tox
- - pip install -U tox
+ # Dependencies
+ - sudo $PYTHON -m pip install -U pytest tox coveralls
 
- # others
- - pip install coveralls --use-wheel
+ # pytest-qt
+ - sudo $PYTHON setup.py develop
 
 before_script:
  - "export DISPLAY=:99.0"
- - "sh -e /etc/init.d/xvfb start"
+ - start-stop-daemon --start --background --exec /usr/bin/Xvfb -- $DISPLAY -screen 0 1024x768x24
  - sleep 3 # give xvfb some time to start
  - "herbstluftwm &"
  - sleep 1
 
 script:
- - python setup.py develop
- - catchsegv coverage run --source=pytestqt setup.py test
- - tox -e lint
+ - sudo catchsegv coverage run --source=pytestqt setup.py test
+ - sudo tox -e lint
```

Comment:
`setuptools` (or `distutils`) would complain about not being able to write some files somewhere, I don't remember where exactly... TBH I didn't investigate it too much. 😅